### PR TITLE
[FSPE-6059] Create E2E Tests for Chi Mobile Navigation functionality

### PR DESCRIPTION
**Pull request by hanyjohny**

### CODE REVIEW

**Comment by hanyjohny** [comment]
Hi @janibolkvadze,

The  PR is ready for your review.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://68cdbbc7a7754d9c6a273ba2f887768c7caba32d-chi-pr.enterprise.ctl.io" target="_blank"> https://68cdbbc7a7754d9c6a273ba2f887768c7caba32d-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1429/ for more details.

File: cypress/integration/chi-js/mobile-nav.spec.js [hunk 1]
```diff
@@ -0,0 +1,175 @@
+const drawerDropDownItem   = '.chi-mobile-nav__list-item-list ul.chi-tabs li';
+
+describe('mobile-nav-menu', function() {
+  before(() => {
+    cy.visit('/tests/js/mobile-nav.html');
+  });
+  describe('mobile-navigation-interaction-trigger button is present', () => {
+    it('mobile-navigation-interaction-trigger button should be visible', () => {
+      cy.get('#mobile-navigation-interaction-trigger', { timeout: 30000 })
```
Comment (janibolkvadze) [inline cypress/integration/chi-js/mobile-nav.spec.js:9]: Timeout is not necessary. Let's remove it please.

File: cypress/integration/chi-js/mobile-nav.spec.js [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+  describe('click on mobile-navigation-interaction-trigger button opens the drawer', () => {
+    it('click mobile-navigation-interaction-trigger opens chi-drawer ', () => {
+      cy.get('#mobile-navigation-interaction-trigger')
+        .click()
+        .wait(1000);
+      cy.get('.chi-drawer')
+        .should('have.class', '-left')
+        .should('have.class', '-active')  
+        .should('be.visible');
+
```
Comment (janibolkvadze) [inline cypress/integration/chi-js/mobile-nav.spec.js:22]:
```suggestion
```

File: cypress/integration/chi-js/mobile-nav.spec.js [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+    it('mobile navigation have logout', () => {
+      cy.get('.chi-mobile-nav__logout')
+        .should('be.visible')
+    });
+    it('mobile navigation has username', () => {
+      cy.get('.chi-mobile-nav__user')
+        .should('be.visible')
+    });
+    
+  });
```
Comment (janibolkvadze) [inline cypress/integration/chi-js/mobile-nav.spec.js:36]:
```suggestion
```

File: cypress/integration/chi-js/mobile-nav.spec.js [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+  describe('open and close of button-site-menu2', () => {
+    it('click on button-site-menu2 open the dropdown items', () => {
```
Comment (janibolkvadze) [inline cypress/integration/chi-js/mobile-nav.spec.js:38]:
```suggestion
    it('click on Drawer Header Dropdown trigger opens the dropdown items', () => {
```

File: cypress/integration/chi-js/mobile-nav.spec.js [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+      cy.get('#button-site-menu2')
+        .click()
+        .wait(1000)
+      cy.get('#dropdown-2-menu')
+        .should('be.visible')
+        .should('have.class','-active');
+    });
+    it('click on button-site-menu2 second time should close the dropdown', () => {
```
Comment (janibolkvadze) [inline cypress/integration/chi-js/mobile-nav.spec.js:46]:
```suggestion
    it('click on Drawer Header Dropdown trigger closes the dropdown', () => {
```

File: cypress/integration/chi-js/mobile-nav.spec.js [hunk 6]
```diff
@@ -0,0 +1,175 @@
...
+      cy.get('.chi-mobile-nav__user')
+        .should('be.visible')
+    });
+    
+  });
+  describe('open and close of button-site-menu2', () => {
+    it('click on button-site-menu2 open the dropdown items', () => {
+      cy.get('#button-site-menu2')
+        .click()
+        .wait(1000)
```
Comment (janibolkvadze) [inline cypress/integration/chi-js/mobile-nav.spec.js:41]: Wait not necessary
```suggestion
```

File: cypress/integration/chi-js/mobile-nav.spec.js [hunk 7]
```diff
@@ -0,0 +1,175 @@
...
+      cy.get('#dropdown-2-menu')
+        .should('be.visible')
+        .should('have.class','-active');
+    });
+    it('click on button-site-menu2 second time should close the dropdown', () => {
+      cy.get('#button-site-menu2')
+        .click()
+        .wait(1000)
```
Comment (janibolkvadze) [inline cypress/integration/chi-js/mobile-nav.spec.js:49]:
```suggestion
```

File: cypress/integration/chi-js/mobile-nav.spec.js [hunk 8]
```diff
@@ -0,0 +1,175 @@
...
+      cy.get('#dropdown-2-menu')
+        .should('not.be.visible')
+        .should('not.have.class','-active');
+    });
+  });
+
+  describe('open and close of button-user-menu2', () => {
```
Comment (janibolkvadze) [inline cypress/integration/chi-js/mobile-nav.spec.js:56]:
```suggestion
  describe('open and close of secondary dropdown of Drawer', () => {
```

File: cypress/integration/chi-js/mobile-nav.spec.js [hunk 9]
```diff
@@ -0,0 +1,175 @@
...
+    it('click on button-user-menu2 open the dropdown items', () => {
```
Comment (janibolkvadze) [inline cypress/integration/chi-js/mobile-nav.spec.js:57]: Same here

File: cypress/integration/chi-js/mobile-nav.spec.js [hunk 10]
```diff
@@ -0,0 +1,175 @@
...
+      cy.get('#button-user-menu2')
+        .click()
+        .wait(1000)
```
Comment (janibolkvadze) [inline cypress/integration/chi-js/mobile-nav.spec.js:60]:
```suggestion
```

File: cypress/integration/chi-js/mobile-nav.spec.js [hunk 11]
```diff
@@ -0,0 +1,175 @@
...
+      cy.get('#dropdown-3-menu')
+        .should('be.visible')
+        .should('have.class','-active');
+    });
+    it('click on button-user-menu2 second time close the dropdown items', () => {
```
Comment (janibolkvadze) [inline cypress/integration/chi-js/mobile-nav.spec.js:65]: Title

File: cypress/integration/chi-js/mobile-nav.spec.js [hunk 12]
```diff
@@ -0,0 +1,175 @@
...
+      cy.get('#button-user-menu2')
+        .click()
+        .wait(1000)
```
Comment (janibolkvadze) [inline cypress/integration/chi-js/mobile-nav.spec.js:68]:
```suggestion
```

File: cypress/integration/chi-js/mobile-nav.spec.js [hunk 13]
```diff
@@ -0,0 +1,175 @@
...
+      cy.get('#dropdown-3-menu')
+        .should('not.be.visible')
+        .should('not.have.class','-active');
+    });
+  });
+
+  describe('Clicking on Tab link should close the drawer', () => {
```
Comment (janibolkvadze) [inline cypress/integration/chi-js/mobile-nav.spec.js:75]:
```suggestion
  describe('Clicking on single level menu item should close the drawer', () => {
```

File: cypress/integration/chi-js/mobile-nav.spec.js [hunk 14]
```diff
@@ -0,0 +1,175 @@
...
+    it('Clicking on Tab link should close the drawer', () => {
+      cy.get('.chi-mobile-nav__first-level-content')
+        .find('.chi-tabs li')
+        .contains('Tab link')
+        .click()
+        .wait(1000)
```
Comment (janibolkvadze) [inline cypress/integration/chi-js/mobile-nav.spec.js:81]:
```suggestion
```

File: cypress/integration/chi-js/mobile-nav.spec.js [hunk 15]
```diff
@@ -0,0 +1,175 @@
...
+      cy.get('.chi-backdrop')
+        .should('have.class','-closed')
+      cy.url().should('include', '#tab-link')
```
Comment (janibolkvadze) [inline cypress/integration/chi-js/mobile-nav.spec.js:84]:
```suggestion
```

File: cypress/integration/chi-js/mobile-nav.spec.js [hunk 16]
```diff
@@ -0,0 +1,175 @@
...
+    });
+  });
+
+  describe('check that click on "X" in menu drawer closes drawer', () => {
+    it('check  drawer closes on "X"', () => {
+      cy.get('#mobile-navigation-interaction-trigger')
+      .click()
+      .wait(1000);
```
Comment (janibolkvadze) [inline cypress/integration/chi-js/mobile-nav.spec.js:92]:
```suggestion
      .click();
```

File: cypress/integration/chi-js/mobile-nav.spec.js [hunk 17]
```diff
@@ -0,0 +1,175 @@
...
+      cy.get('.chi-mobile-nav__first-level-content')
+        .find('.-close')
+        .click()
+        .wait(1000)
```
Comment (janibolkvadze) [inline cypress/integration/chi-js/mobile-nav.spec.js:96]:
```suggestion
```

File: cypress/integration/chi-js/mobile-nav.spec.js [hunk 18]
```diff
@@ -0,0 +1,175 @@
...
+        .parent('.chi-drawer')
+        .should('not.have.class', '-active')
+    });
+  });
+
+  describe('Navigation to the right drawer', () => {
```
Comment (janibolkvadze) [inline cypress/integration/chi-js/mobile-nav.spec.js:102]:
```suggestion
  describe('Opening the second level navigation', () => {
```

File: cypress/integration/chi-js/mobile-nav.spec.js [hunk 19]
```diff
@@ -0,0 +1,175 @@
...
+    it('click on the chi-drawer__subitem-list-trigger link to second level drawer', () => {
+      cy.get('#mobile-navigation-interaction-trigger')
+        .click()
+        .wait(3000);
```
Comment (janibolkvadze) [inline cypress/integration/chi-js/mobile-nav.spec.js:106]:
```suggestion
        .click();
```

File: cypress/integration/chi-js/mobile-nav.spec.js [hunk 20]
```diff
@@ -0,0 +1,175 @@
...
+      cy.get('.chi-mobile-nav__first-level-content')
+        .find('.chi-drawer__subitem-list-trigger')
+        .click()
+        .wait(1000);
```
Comment (janibolkvadze) [inline cypress/integration/chi-js/mobile-nav.spec.js:110]:
```suggestion
        .click();
```

File: cypress/integration/chi-js/mobile-nav.spec.js [hunk 21]
```diff
@@ -0,0 +1,175 @@
...
+        .should('be.visible');
+    });
+  });
+
+  describe('Click on 2nd level menus should open 3rd level drop-downs', () => {
+    it(`check Title A should open Dropdown `, () => {
+      cy.get('#second-level-1')
+        .find('.chi-mobile-nav__list li')
+        .contains('Title A')
+        .as('firstLevelElement');
```
Comment (janibolkvadze) [inline cypress/integration/chi-js/mobile-nav.spec.js:122]: Can we change the declaration name? The clicked element is on the second level already (The first level would be the previous Drawer).

File: cypress/integration/chi-js/mobile-nav.spec.js [hunk 22]
```diff
@@ -0,0 +1,175 @@
...
+      cy.get('@firstLevelElement')
+        .click()
+        .wait(3000)
```
Comment (janibolkvadze) [inline cypress/integration/chi-js/mobile-nav.spec.js:125]:
```suggestion
        .click()
```

File: cypress/integration/chi-js/mobile-nav.spec.js [hunk 23]
```diff
@@ -0,0 +1,175 @@
...
+        .should('have.class', '-expanded') 
+    });
+  });
+
+  describe('check that click on back button  in right drawer ', () => {
+    it('check  right drawer closes on "X"', () => {
+      cy.get('#second-level-1')
+        .find('.chi-button.-back')
+        .click()
+        .wait(1000)
```
Comment (janibolkvadze) [inline cypress/integration/chi-js/mobile-nav.spec.js:139]:
```suggestion
```

File: cypress/integration/chi-js/mobile-nav.spec.js [hunk 24]
```diff
@@ -0,0 +1,175 @@
...
+        .should('not.have.class', '-active');
+    });
+  }); 
+
+  describe('check that click on "X" in right drawer closes drawer', () => {
+    it('check  right drawer closes on "X"', () => {
+      cy.get('.chi-mobile-nav__first-level-content')
+        .find('.chi-drawer__subitem-list-trigger')
+        .click()
+        .wait(1000);
```
Comment (janibolkvadze) [inline cypress/integration/chi-js/mobile-nav.spec.js:150]:
```suggestion
        .click();
```

File: cypress/integration/chi-js/mobile-nav.spec.js [hunk 25]
```diff
@@ -0,0 +1,175 @@
...
+      cy.get('#second-level-1')
+        .find('.-close')
+        .click()
+        .wait(1000)
```
Comment (janibolkvadze) [inline cypress/integration/chi-js/mobile-nav.spec.js:154]:
```suggestion
```

File: cypress/integration/chi-js/mobile-nav.spec.js [hunk 26]
```diff
@@ -0,0 +1,175 @@
...
+        .parent('.chi-drawer')
+        .should('not.have.class', '-active');
+    });
+  });
+
+  describe('backdrop should be present,onclick on backdrop  closes drawer', () => {
+    it('backdrop should be present,onclick on backdrop  closes drawer', () => {
+      cy.get('#mobile-navigation-interaction-trigger')
+        .click()
+        .wait(1000);
```
Comment (janibolkvadze) [inline cypress/integration/chi-js/mobile-nav.spec.js:164]:
```suggestion
        .click();
```

File: cypress/integration/chi-js/mobile-nav.spec.js [hunk 27]
```diff
@@ -0,0 +1,175 @@
...
+      cy.get('.chi-backdrop')
+        .should('be.visible')
+        .should('not.have.class','-closed')
+        .click()
+        .wait(1000)
```
Comment (janibolkvadze) [inline cypress/integration/chi-js/mobile-nav.spec.js:169]:
```suggestion
```

File: test/chi/js/mobile-nav.pug [hunk 28]
```diff
@@ -0,0 +1,105 @@
+---
+title: Mobile Navigation -  component example
+---
+script(src='/js/chi.js')
+button#mobile-navigation-interaction-trigger.chi-button.-icon(data-target='#mobile-navigation-example')
```
Comment (janibolkvadze) [inline test/chi/js/mobile-nav.pug:5]:
```suggestion

button#mobile-navigation-interaction-trigger.chi-button.-icon(data-target='#mobile-navigation-example')
```

File: test/chi/js/mobile-nav.pug [hunk 29]
```diff
@@ -0,0 +1,105 @@
...
+                      li
+                        a.chi-mobile-nav-item-tab.-active Sub tab A
+                      li
+                        a.chi-mobile-nav-item-tab Sub tab B
+                li
+                  a(href='#tab-link')
+                    .chi-mobile-nav__list-title Single level
+
+script.
+  chi.mobilenav(document.querySelector("#mobile-navigation-interaction-trigger"));
```
Comment (janibolkvadze) [inline test/chi/js/mobile-nav.pug:105]:
```suggestion
  chi.mobilenav(document.querySelector("#mobile-navigation-interaction-trigger"));
  
```

**Review by janibolkvadze — CHANGES_REQUESTED** [review]
Looks very good! Let's remove the .wait() rules to save some time and fix the minor linting details. Thank you!

**Comment by allyourbotsarebelongtous** [comment]
Build was ABORTED. See http://jenkins.elasticbox.com/job/chi_pull_request/1450/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build was ABORTED. See http://jenkins.elasticbox.com/job/chi_pull_request/1451/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build was ABORTED. See http://jenkins.elasticbox.com/job/chi_pull_request/1452/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build was ABORTED. See http://jenkins.elasticbox.com/job/chi_pull_request/1453/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build was ABORTED. See http://jenkins.elasticbox.com/job/chi_pull_request/1454/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build was ABORTED. See http://jenkins.elasticbox.com/job/chi_pull_request/1455/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build was ABORTED. See http://jenkins.elasticbox.com/job/chi_pull_request/1456/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build was ABORTED. See http://jenkins.elasticbox.com/job/chi_pull_request/1457/ for more details.

File: cypress/integration/chi-js/mobile-nav.spec.js (hunk 1 above)
Comment (hanyjohny) [reply cypress/integration/chi-js/mobile-nav.spec.js:9]: Hi @janibolkvadze,
 removed the timeout

**Comment by allyourbotsarebelongtous** [comment]
Build was ABORTED. See http://jenkins.elasticbox.com/job/chi_pull_request/1458/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build was ABORTED. See http://jenkins.elasticbox.com/job/chi_pull_request/1459/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build was ABORTED. See http://jenkins.elasticbox.com/job/chi_pull_request/1460/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build was ABORTED. See http://jenkins.elasticbox.com/job/chi_pull_request/1461/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build was ABORTED. See http://jenkins.elasticbox.com/job/chi_pull_request/1463/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build was ABORTED. See http://jenkins.elasticbox.com/job/chi_pull_request/1464/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build was ABORTED. See http://jenkins.elasticbox.com/job/chi_pull_request/1465/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build was ABORTED. See http://jenkins.elasticbox.com/job/chi_pull_request/1466/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build was ABORTED. See http://jenkins.elasticbox.com/job/chi_pull_request/1467/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build was ABORTED. See http://jenkins.elasticbox.com/job/chi_pull_request/1468/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build was ABORTED. See http://jenkins.elasticbox.com/job/chi_pull_request/1469/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build was ABORTED. See http://jenkins.elasticbox.com/job/chi_pull_request/1470/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build was ABORTED. See http://jenkins.elasticbox.com/job/chi_pull_request/1471/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build was ABORTED. See http://jenkins.elasticbox.com/job/chi_pull_request/1472/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build was ABORTED. See http://jenkins.elasticbox.com/job/chi_pull_request/1473/ for more details.

File: cypress/integration/chi-js/mobile-nav.spec.js (hunk 9 above)
Comment (hanyjohny) [reply cypress/integration/chi-js/mobile-nav.spec.js:43]: done

File: cypress/integration/chi-js/mobile-nav.spec.js (hunk 11 above)
Comment (hanyjohny) [reply cypress/integration/chi-js/mobile-nav.spec.js:50]: done

File: cypress/integration/chi-js/mobile-nav.spec.js (hunk 21 above)
Comment (hanyjohny) [reply cypress/integration/chi-js/mobile-nav.spec.js:100]: done

File: cypress/integration/chi-js/mobile-nav.spec.js (hunk 22 above)
Comment (hanyjohny) [reply cypress/integration/chi-js/mobile-nav.spec.js:103]: done

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://a3e5df27df7edf7aba6019cec315b0df7d83f205-chi-pr.enterprise.ctl.io" target="_blank"> https://a3e5df27df7edf7aba6019cec315b0df7d83f205-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1474/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://01b66c092b5fb1daf3b151296b6eea169973d9e5-chi-pr.enterprise.ctl.io" target="_blank"> https://01b66c092b5fb1daf3b151296b6eea169973d9e5-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1475/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build was ABORTED. See http://jenkins.elasticbox.com/job/chi_pull_request/1488/ for more details.

**Comment by hanyjohny** [comment]
> Looks very good! Let's remove the .wait() rules to save some time and fix the minor linting details. Also, a great addition would be testing recovery of Open / Active dropdown on the second level when switching between the dropdowns (Let's discuss this in more details). Thank you!

Hi @janibolkvadze , These testcases are added

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://d6b4395f0ed1250bf9fc46f33d5be8cf1f592cd7-chi-pr.enterprise.ctl.io" target="_blank"> https://d6b4395f0ed1250bf9fc46f33d5be8cf1f592cd7-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1490/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build was ABORTED. See http://jenkins.elasticbox.com/job/chi_pull_request/1491/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://4d1e48aaa859b6be98020ff0526b78cedba21de4-chi-pr.enterprise.ctl.io" target="_blank"> https://4d1e48aaa859b6be98020ff0526b78cedba21de4-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1492/ for more details.

File: cypress/integration/chi-js/mobile-nav.spec.js [hunk 30]
```diff
@@ -0,0 +1,212 @@
...
+    it(`click on  3rd level drop-downs items should close the drawer`, () => {
+      cy.get(drawerDropDownItem)
+        .contains('Sub tab A')
+        .click()
+        .parent('.chi-drawer')
+        .should('not.have.class', '-active');
+    });
+  });
+
+  describe('check that click on back button  in right drawer ', () => {
```
Comment (janibolkvadze) [inline cypress/integration/chi-js/mobile-nav.spec.js:171]:
```suggestion
  describe('Back button functionality ', () => {
```

File: cypress/integration/chi-js/mobile-nav.spec.js [hunk 31]
```diff
@@ -0,0 +1,212 @@
...
+    it('check that click on back button  in right drawer ', () => {
```
Comment (janibolkvadze) [inline cypress/integration/chi-js/mobile-nav.spec.js:172]:
```suggestion
    it('check that click on back button closes second level drawer ', () => {
```

**Review by janibolkvadze — CHANGES_REQUESTED** [review]
Awesome! Let's tweak these 2 texts and it will be good to go. Thank you!

**Comment by allyourbotsarebelongtous** [comment]
Build was ABORTED. See http://jenkins.elasticbox.com/job/chi_pull_request/1503/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://139377e3b0694eb6cc63c95fef9d7da92e6ef842-chi-pr.enterprise.ctl.io" target="_blank"> https://139377e3b0694eb6cc63c95fef9d7da92e6ef842-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1504/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://636cf46e9deec638904d4a1614f33978bf42980b-chi-pr.enterprise.ctl.io" target="_blank"> https://636cf46e9deec638904d4a1614f33978bf42980b-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1506/ for more details.

**Review by janibolkvadze — APPROVED** [review]

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-3y1jlh, https://ops.elasticbox.com/services/instances/i-m2c96j, https://ops.elasticbox.com/services/instances/i-aapld6, https://ops.elasticbox.com/services/instances/i-1imtp8, https://ops.elasticbox.com/services/instances/i-1svbe4, https://ops.elasticbox.com/services/instances/i-ua086j, https://ops.elasticbox.com/services/instances/i-fghr9t

**Pull request merged**